src/app/jobs/pages/jobs/jobs.component.ts

```diff
@@ -111,6 +114,14 @@ export class JobsComponent implements OnInit, AfterViewInit {
     });
   }
 
+  modifyJob(item: JobOffer) {
+    this.dialog.open(JobAddDialogComponent, {
+      width: "550px",
+      height: "500px",
+      data: item,
+    });
+  }
+
```

Comment:
I think the `this.dialog.open` call can be done in the `startEdit` method that already exists in lines 82-84.